CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
-### 3.8.1
+## 3.8.2
 
-#### Fixes
+### Fixes
+
+- # Add coverage-final.json as a possible coverage file during file lookup
```

Comment:
We still don't grab `coverage-summery.json`, correct?

Reply:
We do not afaik